css/style.css

```diff
@@ -1,4 +1,538 @@
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 18px;
+  color: #000;
+  cursor: pointer;
+  padding: 0 6px 0;
+}
+
+.menu__row .menu__item:first-child {
+  border-bottom: 2px solid #e51937;
+  color: #e51937;
+  display: block;
+}
+
+.menu__item:hover {
+  color: #e51937;
+  -webkit-transition: .5s ease-in-out;
+  -o-transition: .5s ease-in-out;
+  transition: .5s ease-in-out;
+}
+
+.header__right_row {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  flex: 0 1 203px;
+  justify-content: space-between;
+  padding: 0 15px 0;
+}
+
+.header__right_row .menu__item {
+  margin: 0 30px 0 0;
+}
+
+.check-menu {
+  display: none;
+  cursor: pointer;
+}
+
+.header__burger {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+}
+
+.burger-line {
+  width: 24px;
+  height: 2px;
+  background-color: #e51937;
+  margin: 2px 0;
+}
+
+.banner {
+  margin: 0 auto;
+  padding: 70px 0 0;
+}
+
+.main-title {
+  font-weight: 600;
+  font-size: 90px;
+  color: #000;
+  text-align: center;
+}
+
+.banner__text {
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 18px;
+  text-align: center;
+}
+
+.banner__img_cover {
+  max-width: 1218px;
+  margin: 0 auto;
+}
+
+.banner__img {
+  width: 100%;
+  height: 100%;
+  object-fit: scale-down;
+}
+
+.test {
+  margin: 30px auto 0;
+}
+
+.test .test__row {
+  max-width: 802px;
   margin: 0 auto;
-  font-family: 'Montserrat', sans-serif; }
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+}
+
+.test__row_right {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  width: 378px;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+}
+
+.test__row_left {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  width: 320px;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+}
+
+.test__column:nth-child(3) {
+  margin: 10px 0 0 0;
+}
+
+.test__item {
+  padding: 0 7px 0 0;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+}
+
+.test__right {
+  padding: 0;
+}
+
+.test__column_testdrive {
+  padding: 10px 0 0;
+}
+
+.test__body {
+  font-weight: 600;
+  font-size: 35px;
+  margin: 0;
+}
+
+.test__body_small {
+  font-size: 20px;
+}
+
+.test__body_bottom {
+  font-size: 12px;
+  line-height: 15px;
+  margin: 1px 0 0;
+}
+
+.test__body_testdrive {
+  font-weight: bold;
+  font-size: 13px;
+  line-height: 24px;
+  text-align: center;
+  text-transform: uppercase;
+  padding: 0 0 0 30px;
+  color: #000;
+  text-shadow: 3px 5px 10px rgba( 0, 0, 0, .33);
```

Comment:
Codacy found an issue: [Unexpected whitespace after "(" in a single-line function (function-parentheses-space-inside)](https://app.codacy.com/gh/WebHeroSchool/chendler_tesla/pullRequest?prid=5804291)